flash

```diff
@@ -202,7 +202,7 @@ while getopts 'hivp:' flag; do
     h) print_usage && exit ;;
     i) print_info && exit ;;
     v) echo -e "\n${YELLOW}fla.sh Current Version:${NC} ${RED}1.1${NC}\n" && exit ;;
-    p) [[ $(command -v $OPTARG 2>&1) ]] && PREVIEWER=$OPTARG || { echo "Unable to find previewer $OPTARG. Exiting..." && exit 1 ; } ;;
+    p) if [[ $(command -v "$OPTARG" 2>&1) ]]; then PREVIEWER=$OPTARG; else echo "Unable to find previewer $OPTARG. Exiting..." && exit 1; fi ;;
```

Comment:
walk me through what this is doing please: 
```sh
command -v "$OPTARG" 2>&1
```

Reply:
`OPTARG` is a variable that stores the argument after the `-p` flag. Note the colon after the p in the getopts while loop. That tells getopts to expect an argument after the `-p` flag.

Specifically what the `command -v` part checks if the argument is installed on the system. Basically it doesn't allow the user to put in whatever they want. They have to put in an installed program.

Reply:
and 
```sh
2>&1
```
is just sending error messages to stdout so user can see the errors, IIRC, right?

Reply:
Yep! It will only print out if there is an error essentially.